src/App.jsx

```diff
@@ -1,14 +1,25 @@
 import React from 'react';
 import './App.css';
+import { Sum } from './components/Sum/Sum';
 
 export const App = () => (
-  <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove commented lines :) */}
-  </>
+  <div>
+    <>
+      <Sum
+        a={2}
+        b={3}
+      />
+      <Sum
+        a={-5}
+        b={5}
+      />
+      <Sum
+        a={10}
+      />
+      <Sum
+        b={5}
+      />
+      <Sum />
+    </>
+  </div>
```

Comment:
empty tag '<>' is used for making wrapper for the few tags, in this case, you added <div> and <> it's too much
so <div> wrapper is redundant